lib/models/message.ts

```diff
@@ -281,4 +293,52 @@ export class MessageModel extends DocumentDbModel<
       query: "SELECT * FROM messages m WHERE (m.fiscalCode = @fiscalCode)"
     });
   }
+
+  public async attachStoredContent(
+    blobService: BlobService,
+    messageId: string,
+    partitionKey: string,
+    message: IMessageWithContent
+  ): Promise<Either<Error, Option<DocumentDb.AttachmentMeta>>> {
+    // this is the attachment id __and__ the filename
+    const blobId = this.getMessageAttachmentName(messageId);
+
+    // store media (attachment) with message content in blob storage
+    const errorOrMessageContent = await upsertBlobFromText(
+      blobService,
+      MESSAGE_BLOB_STORAGE_CONTAINER_NAME,
+      blobId,
+      JSON.stringify({ id: messageId, ...message })
+    );
+
+    if (errorOrMessageContent.isLeft) {
+      return left(errorOrMessageContent.left);
+    }
+
+    // attach the created media to the message identified by messageId and partitionKey
```

Comment:
`media` -> `attachment`

Reply:
actually media and attachment are two different thigs (expecially inside azure documentation).

a media is a blob stored inside azure blob storage. an attachment is some metadata associated to the cosmosdb document that link the media to the document.

so an attachment cannot be stored inside blob storage, as it live inside cosmosdb as metadata.